dts/bindings/video/ovti,ov7670.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: OV7670 CMOS video sensor
+
+compatible: "ovti,ov7670"
+
+properties:
+  reset-gpios:
+    type: phandle-array
+    description: |
+      The RESETn pin is asserted to disable the sensor causing a hard
+      reset.  The sensor receives this as an active-low signal.
```

Comment:
nit. `.  The sensor receives this as an active-low signal.` one extra space here